js/movie_browser.js

```diff
@@ -1,2 +1,19 @@
-// API Docs at: 
-// http://www.omdbapi.com
\ No newline at end of file
+// API Docs at:
+// http://www.omdbapi.com
+
+$(document).ready(()=>{
+  let input = 0
+  $.ajax({
+    type: 'GET',
+    dataType: 'json',
+    url: "/movie_browser"
```

Comment:
so you actually need to be setting the url to the omdb api for this to work. and you'd need to add a template literal to the search parameter in the url (which you can find in the ombd docs) that would capture the value (.val()) that a user would input into the form.